FabricExample/App.js

```diff
@@ -1,3 +1,4 @@
+import 'react-native-gesture-handler';
```

Comment:
Why do we import GH here?

Reply:
That's how react-native-gesture-handler should be used, according to the installation steps from the [docs](https://docs.swmansion.com/react-native-gesture-handler/docs/fundamentals/installation). Also, I had to add this dependency here, since react-navigation requires having react-native-gesture-handler for using navigator from `@react-navigation/stack`. I know we won't mostly use this stack, as we want to mainly test native-stack of course, but in case of trying to use `createStackNavigator` let's leave this import here and disallow to throw weird exception of not bundled gesture-handler 😄